lib/scss_lint/runner.rb

```diff
@@ -15,9 +17,10 @@ def initialize(config)
     # @param files [Array<Hash>] list of file object/path hashes
     def run(files)
       @files = files
-      @files.each do |file|
+      result = @files.each do |file|
```

Comment:
What are you trying to do here? `result` will be set to `@files` since you're using `each`...